opal/models.py

```diff
@@ -122,6 +122,52 @@ def _get_field_type(cls, name):
         )
 
     @classmethod
+    def get_human_readable_type(cls, field):
```

Comment:
This is ++ useful, presumably at some point for the in-line documentation on search / extract pages?

I'm a little concerned about just how much we hoist onto our models though... an alternative implementation here would be to have this as a function that takes the actual field in e.g. `opal.models.schema` ?

As a minimum I think we'd want to document the method/function existing, and if it is going to be a classmethod that takes a string field name then we might rename the argument?